.env.example

```diff
@@ -6,7 +6,7 @@ APP_TIMEZONE=America/New_York
 APP_CLEAR_TASKLOG=720
 APP_DELETE_MINUTES=10
 APP_ENVIRONMENT_ONLY=true
-LANG_MUST_BE_GLOBAL=false
+APP_LOCALE_IS_GLOBAL=false
```

Comment:
As you're at it: Could you also add `APP_LOCALE=en` to the .env.example? Then this setting isn't that out-of-context

Reply:
Sure